imdb_django/models.py

```diff
@@ -0,0 +1,134 @@
+# Create your models here.
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        app_label = 'imdb_django'
+
+    def __str__(self):
+        return self.name
+
+
+class Title(models.Model):
+    t_const = models.CharField(max_length=20, primary_key=True)
+    title_Type = models.CharField(max_length=50)
+    primary_Title = models.TextField()
+    original_Title = models.TextField()
+    is_Adult = models.BooleanField()
+    start_Year = models.CharField(max_length=20, blank=True)
+    end_Year = models.CharField(max_length=20, blank=True)
+    runtime_Minutes = models.CharField(max_length=20, blank=True)
+    genres = models.ManyToManyField(Genre, related_name='genres')
+
+    class Meta:
+        app_label = 'imdb_django'
+
+    def __str__(self):
+        return f"({self.t_const}, {self.primary_Title})"
+
+
+class TitleAka(models.Model):
+    t_const = models.ForeignKey(Title, related_name='Aka', on_delete=models.CASCADE)
+    ordering = models.PositiveIntegerField()
+    title = models.TextField()
+    region = models.CharField(max_length=10)
+    language = models.CharField(max_length=10)
+    types = models.CharField(max_length=255)
+    attributes = models.CharField(max_length=255)
+    is_Original_Title = models.BooleanField()
+
+    class Meta:
+        app_label = 'imdb_django'
+
+    def __str__(self):
+        return f"({self.t_const}, {self.title})"
+
+
+class TitleCrew(models.Model):
+    t_const = models.ForeignKey(Title, related_name='crew', on_delete=models.CASCADE)
+    directors = models.ManyToManyField("Name", related_name="directed_titles")
+    writers = models.ManyToManyField("Name", related_name="written_titles")
+
+    class Meta:
+        app_label = 'imdb_django'
+
+    def __str__(self):
+        return f"({self.directors}, {self.writers})"
+
+
+class TitleEpisode(models.Model):
+    t_const = models.OneToOneField(Title, primary_key=True, related_name='episodes', on_delete=models.CASCADE)
+    title = models.ForeignKey(
+        Title, related_name="episodes", on_delete=models.CASCADE
+    )
+    season_Number = models.PositiveIntegerField(unique=True)
+    episode_Number = models.PositiveIntegerField(unique=True)
```

Comment:
Instead of adding unique to each field we need [unique_togather](https://docs.djangoproject.com/en/4.2/ref/models/options/#unique-together)

For example: a combination of season and episode cannot repeat

Episode list:
s01e01
s01e02
s01e03
s01e04
s01e01 <--- episode repeating
s01e04 <--- episode repeating
s01e05
s01e06
s01e07
s01e08